src/Deftype.hs

```diff
@@ -428,6 +434,7 @@ binderForStrOrPrn typeEnv env insidePath structTy@(StructTy (ConcreteNameTy type
             (concreteStr typeEnv env structTy (memberXObjsToPairs membersXObjs) strOrPrn)
             ("converts a `" ++ typeName ++ "` to a string.")
         )
+binderForStrOrPrn _ _ _ _ _ _ = error "binderforstrorprn"
```

Comment:
I want to see the face of a person who gets "binderforstrorprn" as an error message :D